gnomad_constraint/utils/constraint.py

```diff
@@ -459,18 +465,21 @@ def apply_models(
 
     # Compute observed and possible variant counts.
     ht = create_observed_and_possible_ht(
-        exome_ht,
-        context_ht,
-        mutation_ht,
-        max_af,
-        keep_annotations,
-        pops,
-        grouping,
-        obs_pos_count_partition_hint,
+        exome_ht=exome_ht,
+        context_ht=context_ht,
+        mutation_ht=mutation_ht,
+        max_af=max_af,
+        keep_annotations=keep_annotations,
+        pops=pops,
+        downsamplings=downsamplings,
+        grouping=grouping,
+        partition_hint=obs_pos_count_partition_hint,
         filter_coverage_over_0=True,
         transcript_for_synonymous_filter=None,
     )
 
+    ht = ht.checkpoint(new_temp_file(prefix="oe_ht", extension="ht"))
```

Comment:
Add a comment about why this checkpoint it needed/helpful

Reply:
nm, it's checkpointed in the function